src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -50,32 +50,302 @@ Notifier::~Notifier (void)
   LM_T(LmtNotImplemented, ("Notifier destructor is not implemented"));
 }
 
+
+
+/* ****************************************************************************
```

Comment:
Notifier.cpp is focused in Notifier class method implementation. Maybe new methods substitude(), etc. should be in a separate .cpp/.h.

Not really sure if this is the best alternative... just proposing in case you haven't realize.